aggsender/validator/config.go

```diff
@@ -40,9 +47,34 @@ type PPConfig struct {
 	RequireOneBridgeInPPCertificate bool `mapstructure:"RequireOneBridgeInPPCertificate"`
 }
 
+type FEPConfig struct {
+	// SovereignRollupAddr is the address of the sovereign rollup contract on L1
+	SovereignRollupAddr ethCommon.Address `mapstructure:"SovereignRollupAddr"`
+	// RequireNoFEPBlockGap is true if the AggSender should not accept a gap between
+	// lastBlock from lastCertificate and first block of FEP
+	RequireNoFEPBlockGap bool `mapstructure:"RequireNoFEPBlockGap"`
```

Comment:
Maybe now that it is grouped under the `FEPConfig`, maybe it can be renamed to exclude the FEP from the param name, wdyt?


```suggestion
	RequireNoBlockGap bool `mapstructure:"RequireNoBlockGap"`
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/876/commits/cabebef893f28b179d2744345de554907013307b